src/System.Private.CoreLib/src/System/Threading/ClrThreadPool.WorkerThread.cs

```diff
@@ -22,11 +22,11 @@ private static class WorkerThread
             
             private static void WorkerThreadStart()
             {
-                // TODO: Event: Worker Thread Start event
+                ClrThreadPoolEventSource.Log.WorkerThreadStart(ThreadCounts.VolatileReadCounts(ref ThreadPoolInstance._separated.counts).numExistingThreads);
                 RuntimeThread currentThread = RuntimeThread.CurrentThread;
                 while (true)
                 {
-                    // TODO: Event:  Worker thread wait event
+                    ClrThreadPoolEventSource.Log.WorkerThreadWait(ThreadCounts.VolatileReadCounts(ref ThreadPoolInstance._separated.counts).numExistingThreads);
```

Comment:
I've added another call to this at the end of the loop below to ensure we are catching all the logs.